Common.cmake

```diff
@@ -128,6 +129,12 @@ if(NOT TARGET ${PROJECT_NAME}-all)
   set_target_properties(${PROJECT_NAME}-all PROPERTIES FOLDER ${PROJECT_NAME})
 endif()
 
+add_custom_target(${PROJECT_NAME}-install
+  ${CMAKE_COMMAND} -P ${PROJECT_BINARY_DIR}/cmake_install.cmake
+  DEPENDS ${PROJECT_NAME}-all)
+set_target_properties(${PROJECT_NAME}-install PROPERTIES
+  EXCLUDE_FROM_DEFAULT_BUILD ON FOLDER ${PROJECT_NAME}/doxygen)
```

Comment:
What does the ON FOLDER do? I don't find the documentation that explains it.
Fix the C&P reference to doxygen in the path.